### DISCUSSION TITLE
Is there any way to restart your chromebook without updating?

### DISCUSSION THREAD

**Discussion by Fallenmoon8080** [3kh0/ext-remover#108] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I was wondering if it could be possible to light reboot or something like that. Just wanted to know.

**Comment by synapt0** [comment, marked as answer]
Yes, but that is only if you are allowed to turn auto-updates off. If your Chromebook can't do that, then I'm afraid no.
